core/templates/dev/head/pages/exploration_editor/editor_tab/StateSolution.js

```diff
@@ -60,35 +56,27 @@ oppia.controller('StateSolution', [
           stateInteractionIdService.savedMemento,
           explorationStatesService.getInteractionCustomizationArgsMemento(
             editorContextService.getActiveStateName()),
-          'currentInteractionHtml'));
+          'currentInteractionHtmlForSolutionEditor'));
```

Comment:
Pull this out into a SOLUTION_EDITOR_FOCUS_LABEL constant to use in both the HTML and the JS.